pkg/mcs/tso/server/server.go

```diff
@@ -448,14 +448,15 @@ func (s *Server) startServer() (err error) {
 	}
 
 	// Server has started.
-	entry := &discovery.ServiceRegistryEntry{ServiceAddr: s.cfg.ListenAddr}
+	entry := &discovery.ServiceRegistryEntry{ServiceAddr: s.cfg.AdvertiseListenAddr}
```

Comment:
shall we remove http:// or https://? Probably doesn't matter

Reply:
I think it doesn't matter unless we have a strong inclination about it.

Reply:
bette remove http://, because register use path.Join, and path.Join("aaa", "http://bbb/")返回"aaa/**http:/**bbb" in which it contains http:/ instead of http://

Reply:
do we need to add related test?